coding/python_style_guide.rst

```diff
@@ -15,6 +15,29 @@ The :doc:`intro` provides the overarching Coding Standards policy applicable to
 
 .. _style-guide-py-intro:
 
+0. Python Version
+=================
+
+.. _style-guide-py-version:
+
+All DM Python code MUST work with Python 3
+------------------------------------------
+
+All the Python code written by LSST Data Management must be runnable using Python 3.
+Python 2 will cease to be supported before LSST is operational (:pep:`373`).
+The current baseline version is Python 3.5 or above.
+
+DM Python library code with an external user base MUST support Python 2.7 and 3.x
```

Comment:
@fred3m @jdswinbank I do state 2.7 here. So I think that's fine. The 3.x here is to stop duplicating the explicit version number stated above.